### ISSUE TITLE
Issue with k8s.io/docs/tutorials/kubernetes-basics/expose/expose-intro/

### ISSUE THREAD

**Issue by patrakov** [kubernetes/website#19625]
**This is a Bug Report**

**Problem:**
There is a side note:

> You can create a Service at the same time you create a Deployment by using --expose in kubectl.

However, `kubectl create deployment --help` does not list `--expose` as a valid flag, and in fact does not accept it.

**Proposed Solution:**

Remove the side note? Or, if I am supposed to pass --expose elsewhere, reword it to avoid this misunderstanding?

**Page to Update:**

https://kubernetes.io/docs/tutorials/kubernetes-basics/expose/expose-intro/

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-11T18:14:22Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-11T18:07:13Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by tengqm** [comment]
Looks like this is fixed. Feel free to reopen if it is not.
/close
